components/ws-manager-bridge/src/bridge.ts

```diff
@@ -462,24 +466,11 @@ export class WorkspaceManagerBridge implements Disposable {
             instance.stoppedTime = new Date().toISOString();
             promises.push(this.workspaceDB.trace({}).storeInstance(instance));
             promises.push(this.onInstanceStopped({}, ri.workspace.ownerId, instance));
-            promises.push(this.stopPrebuildInstance(ctx, instance));
+            promises.push(this.prebuildUpdater.stopPrebuildInstance(ctx, instance));
```

Comment:
@geropl I don't think this does anything useful. Looks to me like it would update the workspace instance that got set to 'stopped' a few lines above now to 'aborted'. Am I missing something?

Reply:
> Looks to me like it would update the workspace instance that got set to 'stopped' a few lines above now to 'aborted'. Am I missing something?

It updates the `PrebuiltWorkspace` that's corresponding that `WorkspaceInstance` to `aborted`, which would otherwise not bet updated.

In what context are you looking into this? Can I help?

Reply:
> It updates the PrebuiltWorkspace that's corresponding that WorkspaceInstance to aborted, which would otherwise not bet updated.

Why does it do that? And how can a prebuild not be finished when there is a workspace based on it?

> In what context are you looking into this? Can I help?

Just reading the code and trying to make sense of it 😇

Reply:
Ah, got it now. Forget it 😁

Reply:
What confused me is that the method is called `stopPrebuildInstance` while it only aborts the PrebuildWorkspace in the database.

Reply:
All good :wink: